Signal/src/call/CallService.swift

```diff
@@ -144,8 +144,10 @@ protocol CallServiceObserver: class {
                 if let oldValue = oldValue {
                     DeviceSleepManager.sharedInstance.removeBlock(blockObject: oldValue)
                 }
+                stopAnyActiveCallTimer()
                 if let call = call {
                     DeviceSleepManager.sharedInstance.addBlock(blockObject: call)
+                    self.startActiveCallTimer()
```

Comment:
I feel like this slightly more strongly ensure the invariant "there is a timer IFF there is a call."